data/incipias/incipias first contact.txt

```diff
@@ -158,6 +160,7 @@ mission "Incipias: The Derelict"
 		to despawn
 			has "incipias: helped the derelict"
 		on assist
+			log `Assisted a stranded alien vessel in an uninhabited system, whose inhabitants appeared similar to floating octopi or jellyfish. Could not communicate with them due to the language barrier, but managed to give them the repair equipment they needed to repair their ship. However, they lacked a hyperdrive and so were unable to travel back to their home system.`
```

Comment:
```suggestion
			log `Assisted a stranded alien vessel in an uninhabited system with crew members that appear similar to floating octopi or jellyfish. Could not communicate with them due to the language barrier, but managed to give them the repair equipment they needed to repair their ship. However, they lacked a hyperdrive and so were unable to travel back to their home system.`
```
This is a better example of more written-out logs. Though, having "inhabitants" read right after "uninhabited" is a bit jarring. Changed it so it specifically refers to the crew you met.